config/grad.uiowa.edu/field.storage.node.field_person_scholar_srop_year.yml

```diff
@@ -0,0 +1,87 @@
+uuid: ef77246e-be0d-4a95-a69a-d5923ec83477
+langcode: en
+status: true
+dependencies:
+  module:
+    - node
+    - options
+id: node.field_person_scholar_srop_year
+field_name: field_person_scholar_srop_year
+entity_type: node
+type: list_string
+settings:
+  allowed_values:
+    -
+      value: '2000'
+      label: '2000'
+    -
+      value: '2001'
+      label: '2001'
+    -
+      value: '2002'
+      label: '2002'
+    -
+      value: '2003'
+      label: '2003'
+    -
+      value: '2004'
+      label: '2004'
+    -
+      value: '2005'
+      label: '2005'
+    -
+      value: '2006'
+      label: '2006'
+    -
+      value: '2007'
+      label: '2007'
+    -
+      value: '2008'
+      label: '2008'
+    -
+      value: '2009'
+      label: '2009'
+    -
+      value: '2010'
+      label: '2010'
+    -
+      value: '2011'
+      label: '2011'
+    -
+      value: '2012'
+      label: '2012'
+    -
+      value: '2013'
+      label: '2013'
+    -
+      value: '2014'
+      label: '2014'
+    -
+      value: '2015'
+      label: '2015'
+    -
+      value: '2016'
+      label: '2016'
+    -
+      value: '2017'
+      label: '2017'
+    -
+      value: '2018'
+      label: '2018'
+    -
+      value: '2019'
+      label: '2019'
+    -
+      value: '2020'
+      label: '2020'
+    -
+      value: '2021'
+      label: '2021'
+  allowed_values_function: ''
```

Comment:
I think this is going to need an `allowed_values_function` to be able to return a list of years that is dynamic, so that we don't have to update this field next year.

Reply:
OH! I'll get started on that! Thank you!

Reply:
implemented and tested. Learned something new today and it's not even 9 yet! Thank you 💯  Is how I went about it okay?